### PR TITLE
ci: Initial support for building a dev BOSH release

### PR DESCRIPTION
**Pull request by evandbrown**
This change adds a simple Concourse pipeline that creates a BOSH release and its SHA1 and uploads them both to Google Cloud Storage. The accompanying README documents setting up the required infrastructure with Terraform.

### CODE REVIEW

File: ci/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Deploy CI Infrastructure
+
+1. Export your project ID, a prefix, and the pipeline (develop or prod):
+
+  ```
+  projectid=<REPLACE_WITH_YOUR_PROJECT_ID>
+  prefix=gcp-tools
```
Comment (johnsonj) [inline ci/README.md]: did you consider gcp-tools-boshrelease to match repo name? it is a bit wordy

File: ci/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+1. Edit `credentials-*.yml` and replace ` {{service_account_key_json}}` with the contents of `cat /tmp/${pipeline}-${prefix}.key.json`
+
+# Deploy Pipeline
+
+1. Login to Concourse:
+
+  ```
+  fly --target lambr login \
+    --team-name gcp-tools-boshrelease \
+    --concourse-url https://107.178.243.59/ -k
```
Comment (johnsonj) [inline ci/README.md]: is this specific to our pipeline?

File: ci/pipeline-develop.yml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+      - put: gcp-tools-boshrelease-artifacts-sha1
+        params: {file: candidate/*.tgz.sha1}
+
+resources:
+  - name: gcp-tools-boshrelease-in
+    type: git
+    source:
+      uri: https://github.com/evandbrown/gcp-tools-boshrelease.git
+      branch: pr-add-ci-pipeline
```
Comment (johnsonj) [inline ci/pipeline-develop.yml]: probably want to change before merge

File: ci/README.md (hunk 2 above)
Comment (evandbrown) [inline ci/README.md]: Yes. Good catch!
